src/OrleansRuntime/Core/InsideRuntimeClient.cs

```diff
@@ -580,23 +578,11 @@ public void ReceiveResponse(Message message)
             }
         }
 
-        public Logger AppLogger
-        {
-            get { return appLogger; }
-        }
+        public Logger AppLogger => appLogger;
```

Comment:
In the future, it would help to group such non-essential changes in a separate commit to simplify reviewing. It's pretty easy to do with SourceTree.